health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/WorkflowCommentPopUp.js

```
@@ -35,6 +32,8 @@ const WorkflowCommentPopUp = ({ onClose, heading, submitLabel, url, requestPaylo
         const key = Object.keys(payloadObject)?.[0]; // Get the first key in the payloadObject
         const value = payloadObject[key]; // Get the value associated with that key
 
+        const finalComment = comment && comment.trim() ? comment : null;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)